### PR TITLE
Add JobSet presubmit for unit tests

### PR DESCRIPTION
**Pull request by danielvegamyhre**
Add JobSet presubmit for unit tests

Project repo: https://github.com/kubernetes-sigs/jobset

cc @ahg-g

### CODE REVIEW

File: config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/jobset:
+  - name: pull-jobset-test-unit-main
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-scheduling
```
Comment (ahg-g) [inline config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml:8]: should be in sig-apps

Comment (danielvegamyhre) [reply config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml]: Done

File: config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+presubmits:
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml:1]: please consider adding an OWNERS file scoped to kubernetes-sigs/jobset so jobset maintainers can self approve small changes to this.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

File: config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml (hunk 2 above)
Comment (danielvegamyhre) [reply config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml:1]: @BenTheElder just added an OWNERS file which accidentally removed your LGTM, sorry about that

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml:1]: NBD, lgtm anyone in the org can add as well :-)

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

File: config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml (hunk 2 above)
Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml:1]: Thank you!

**Pull request merged**